<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tela_cadastrar_produto</class>
 <widget class="QWidget" name="tela_cadastrar_produto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>236</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cadastrar Produto</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icones/icone_produto.png</normaloff>../icones/icone_produto.png</iconset>
  </property>
  <property name="windowFilePath">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label_descricao">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string notr="true">Descrição:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_marca">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string notr="true">Marca:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tamanho">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string notr="true">Tamanho:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_quantidade">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string notr="true">Quantidade:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_preco_compra">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string notr="true">Preço de compra:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="texto_descricao">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="maxLength">
    <number>50</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="texto_marca">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="maxLength">
    <number>50</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="texto_tamanho">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="maxLength">
    <number>50</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_box_quantidade">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="maximum">
    <number>99999</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="double_spin_box_preco_compra">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="maximum">
    <double>9999.989999999999782</double>
   </property>
  </widget>
  <widget class="QPushButton" name="botao_login">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string notr="true">Cadastrar Produto</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
